contrib/checkbox-provider-ce-oem/series_uc20/snap/snapcraft.yaml

```
@@ -0,0 +1,57 @@
+name: checkbox-ce-oem
+summary: Checkbox CE OEM and public providers
+description: "Checkbox CE OEM and public providers"
+confinement: strict
+grade: stable
+
+version: '0.12-focal'
```

How about starting the version from `1.0`?
```suggestion
version: '1.0-focal'
```